### ISSUE TITLE
[MU4 Issue] Accessibility: menus do not read their names (Windows/Linux)

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#12846]
**Describe the bug**
Upon pressing Alt and navigating the menu bar, the names of the menus (File, Edit, etc) are not read by screen reader

**To Reproduce**
1. Start screen reader (tested with NVDA and Orca)
2. Press Alt to focus on the menu bar
3. Use left and right cursor keys to navigate the menus

**Result**
Nothing is read at all

**Expected behavior**
The name of each menu should be read (File, Edit, etc)

**Platform information**
 - OS: Windows, Linux

**Additional context**
In MuseScore 3, Alt alone did not focus on the menu bar; you need Alt+F or other direct mnemonic to open a specific menu.  I think it's an improvement that Alt focuses on the menu bar without needing know any specific menmonic.  But, it's not very useful if the names of the menus don't read!

On Windows, it appears that Tab may also move focus to the menu (after the bottom toolbar), but still, nothing is read aloud.  On Linux, Tab goes directly from the bottom toolbar to the Home/Score/Publish toolbar.  That's actually more what I expect, I think.

**Comment by DmitryArefiev** [comment]
Fixed in #13009 but only on Windows side.

**STILL OCCURS** on **Linux**

**Comment by DmitryArefiev** [comment]
Fixed in #13347
